docs/rgw_setup.md

```diff
@@ -0,0 +1,136 @@
+
+[keystone_all]
+keystone_host
+```
+
+### Preset variables
+rpc-ceph has already assigned some ceph-ansible variable in the file **(playbooks/group_vars/rgws.yml)[https://raw.githubusercontent.com/rcbops/rpc-ceph/master/playbooks/group_vars/rgws.yml]**  If special needs are required for a specific customer's requirements these variable can be changed or overridden
+
+Make changes these lines from the file **playbooks/group_vars/rgws.yml** because they are used for integration with OpenStack
+```
+# Replace these 
+radosgw_keystone_service_name: "swift"
+radosgw_keystone_service_description: "Swift Service"
+# with this
+# Replace these 
+radosgw_keystone_service_name: "ceph"
+radosgw_keystone_service_description: "Ceph Object Service"
+```
+
+### Make sure you have each of these variables set in your vars file
+* public_network
+* internal_lb_vip_address
+* service_region
+* radosgw_keystone **(set to True)**
+* keystone_service_adminurl
+* radosgw_keystone_admin_user
+* radosgw_keystone_admin_password
+* radosgw_keystone_admin_tenant
+* What from this?   https://github.com/rcbops/rpc-ceph/blob/master/playbooks/ceph-keystone-rgw.yml
+
+### Run the deployment
+```bash
+ceph-ansible -i <link to your inventory file> playbooks/deploy-ceph.yml -e@<link to your vars file>
+```
+
+
+## Installation of rgw with Openstack KeyStone and OpenStack Ansible
+To Install rgw and connect it to a Stand Alone instance of OpenStack Keystone.   Please refer to other documentation for the RPC guidelines for installing and setting up OpenStack Keystone.   This ceph rgw guide assumes you have an working OpenStack Keystone system in place
+
+### To install rgw addtions to the inventory file.   
+
+Add the **rgw** group with a list of hosts into the ceph environment's inventory file. 
+For example:
+```bash
+[rgw]
+ceph-rgw01
+ceph-rgw02
+ceph-rgw03
+
+[keystone_all]
+keystone_host
+```
+
+### Preset variables
+rpc-ceph has already assigned some ceph-ansible variable in the file **(playbooks/group_vars/rgws.yml)[https://raw.githubusercontent.com/rcbops/rpc-ceph/master/playbooks/group_vars/rgws.yml]**  If special needs are required for a specific customer's requirements these variable can be changed or overridden
+
+### Make sure you have each of these variables set in your vars file
+* public_network
+* internal_lb_vip_address
+* service_region
+* radosgw_keystone **(set to True)**
+* keystone_service_adminurl
+* radosgw_keystone_admin_user
+* radosgw_keystone_admin_password
+* radosgw_keystone_admin_tenant
+* What from this?   https://github.com/rcbops/rpc-ceph/blob/master/playbooks/ceph-keystone-rgw.yml
+
+### Run the deployment
+```bash
+ceph-ansible -i <link to your inventory file> playbooks/deploy-ceph.yml -e@<link to your vars file>
+```
+
+
+## Load Balancers and SSL Certs
+A load balancer should be deployed prior to ceph rgw install and you have VIP.  Currently RPC terminates SSL in the load balancer so the Ceph Object Store does not need anything for SSL set
```

Comment:
If theyre using an f5 thats ok, if they're using haproxy we may need to explain that a bit more.
It's a bit trickier to setup the vip for ceph in haproxy because it isn't integrated in the RPC-O inventory, but I think we can get around that.